README.md

```diff
@@ -33,95 +32,113 @@ implements the `SpamProtector` interface. More on that below.
 
 ## Configuring
 
-After installing this module and a protector of your choice (i.e mollom) you'll 
-need to rebuild your database through `dev/build` and set the default protector 
+After installing this module and a protector of your choice (i.e mollom) you'll
+need to rebuild your database through `dev/build` and set the default protector
 via SilverStripe's config system. This will update any Form instances that have
 spam protection hooks with that protector.
 
 *mysite/_config/spamprotection.yml*
 
-	---
-	name: spamprotection
-	---
-	FormSpamProtectionExtension:
-	  default_spam_protector: MollomSpamProtector
+```yaml
+---
+name: spamprotection
+---
+SilverStripe\SpamProtection\Extension\FormSpamProtectionExtension:
+  default_spam_protector: MollomSpamProtector
+```
 
 To add spam protection to your form instance call `enableSpamProtection`.
-	
-	// your existing form code
-	$form = new Form( .. );
-	$form->enableSpamProtection();
 
-The logic to perform the actual spam validation is controlled by each of the 
-individual `SpamProtector` implementation since they each require a different 
+```php
+// your existing form code
+$form = new Form(/* .. */);
+$form->enableSpamProtection();
+```
+
+The logic to perform the actual spam validation is controlled by each of the
+individual `SpamProtector` implementation since they each require a different
```

Comment:
[Minor] Consider rewording to either
_The logic to perform the actual spam validation is controlled by each individual `SpamProtector` implementation ..._
or
_The logic to perform the actual spam validation is controlled by each of the individual `SpamProtector` implementations ... _